### ISSUE TITLE
wrong flip

### ISSUE THREAD

**Issue by terrypang** [nicolov/segmentation_keras#21]
Here, x is a single image array, not a batch data. During flip, the row_index should be 0, and the col_index should be 1.

```python
def transform(x: np.array, params: TransformParams) -> np.array:
    if params.do_hor_flip:
        x = flip_axis(x, 2)
    if params.do_vert_flip:
        x = flip_axis(x, 1)

https://github.com/fchollet/keras/blob/master/keras/preprocessing/image.py#L538
> # x is a single image, so it doesn't have image number at index 0

**Comment by nicolov** [comment]
Thanks for the catch, I'll fix over the weekend

**Comment by nicolov** [comment]
Thanks, I've fixed the issue in 3a1ea52
